acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -85,7 +85,7 @@
           sleep $sleep_for
           url='https://#{master}:8140/puppet/v3/catalog/#{agent_cert}?environment=production'
           echo "Curling: $url"
-          curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
+          env PATH='#{agent[:privatebindir]}:${PATH}' curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
```

Comment:
`privatebindir` and `puppetbindir` are one of the few hardcoded beaker defaults as the location of the binaries depends on the `type` of acceptance and the unix vs windows host, e.g. `git` installs things in /usr/bin (mostly), `aio` installs things in `/opt/puppetlabs/puppet/bin`, etc.